### PR TITLE
(#19) - Improve tests for TgzArchive

### PR DESCRIPTION
**Pull request by victornoel**
This is for #19.

Apparently the test was already there, so I added another one and corrected the test file name as well as made a method private since it wasn't meant to be exposed.

### CODE REVIEW

File: src/test/java/com/artipie/helm/TgzArchiveTest.java [hunk 1]
```diff
@@ -56,6 +56,19 @@ public void nameIdentifiedCorrectly() throws IOException {
         );
     }
 
+    @Test
+    public void sizeHasCorrectValue() throws IOException {
+        final Path file = Paths.get("./src/test/resources/tomcat-0.4.1.tgz");
+        MatcherAssert.assertThat(
+            new TgzArchive(
+                Files.readAllBytes(
+                    file
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/helm/TgzArchiveTest.java:65]: @victornoel let's inline this param with the `Files.readAllBytes` call to clean up the formatting a bit.

Comment (victornoel) [reply src/test/java/com/artipie/helm/TgzArchiveTest.java:65]: @carlosmiranda `file` is used in two places in this test

Comment (carlosmiranda) [reply src/test/java/com/artipie/helm/TgzArchiveTest.java:65]: @victornoel I'm just referring to the formatting. Let's collapse lines 64-66 into one line.

Comment (victornoel) [reply src/test/java/com/artipie/helm/TgzArchiveTest.java:65]: @carlosmiranda ok

Comment (victornoel) [reply src/test/java/com/artipie/helm/TgzArchiveTest.java]: @carlosmiranda it's done

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by 0crat** [comment]
Job was finished in 19 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by Sammers21** [comment]
> This is for #19.
> 
> Apparently the test was already there, so I added another one and corrected the test file name as well as made a method private since it wasn't meant to be exposed.

@victornoel, it was meant to be exposed

**Comment by victornoel** [comment]
@Sammers21 ok, since it was not used, maybe later it will be reexposed when needed, no harm done :)

**Pull request merged**